.circleci/config.yml

```diff
@@ -52,12 +101,19 @@ jobs:
 
 workflows:
   version: 2.1
-  build_and_deploy:
+  build_and_test:
     jobs:
       - build
-      - deploy:
+      - tests-with-coverage:
           requires:
             - build
+      - tests-on-windows:
+          requires:
+            - build
+      - deploy:
+          requires:
+            - tests-with-coverage
+            - tests-on-windows
           filters:
```

Comment:
We don't need the branch filters for build and test right? We just want the filters to apply to deploying

Reply:
Yea thats what I thought this said.
Deploy requires these two builds and only runs on master.